docs/datascience/jupyter-kernel-options.md

```diff
@@ -0,0 +1,43 @@
+---
+Order: 7
+Area: datascience
+TOCTitle: Jupyter Kernel Options in VS Code
+ContentId: 3b6da7e6-c449-4c62-a019-9202412aac04
+PageTitle: Jupyter Kernel Options in VS Code
+DateApproved: 11/21/2022
+MetaDescription: Descriptions, tips, and tricks on kernel selection options when working with Jupyter Notebooks in Visual Studio Code.
+MetaSocialImage: images/tutorial/social.png
+---
+
+# Jupyter Kernel Options in VS Code <!-- omit in toc -->
+
+The VS Code notebooks’ kernel picker lists the following categories of connection methods to run the code in your Jupyter notebook:
+- [Jupyter Kernels](#jupyter-kernels)
+- [Python Environments](#python-environments)
+- [Existing Jupyter Server](#existing-jupyter-server)
+
+By default, we will recommend the one you’ve previously used with your notebook, but you can choose to connect to any other Jupyter kernels as shown below.
+
+## Jupyter Kernels
+
+The **Jupyter Kernels** category lists all Jupyter kernels that VS Code detects in the context of the compute system it’s operating in (your desktop, Codespaces, remote server, etc.). Each Jupyter kernels has Jupyter [kernel specification](https://jupyter-client.readthedocs.io/en/stable/kernels.html#kernel-specs), or Jupyter kernelspec, which contains a JSON file (i.e., `kernel.json`) with details about the kernel—name, description, CLI information required to launch a process as a kernel.
+
+## Python Environments
+
+The **Python Environments** category lists the Python environments that VS Code detects from the compute system it’s operating in (your desktop, Codespaces, remote server, etc.). It shows all Python environments grouped by type (e.g., conda, venv)—whether the [IPyKernel](https://ipython.readthedocs.io/en/stable/install/kernel_install.html) is installed or not.
+
+   > **Note**: You **__do not__** need to install [jupyter](https://pypi.org/project/jupyter/) into the Python environment you want to use. Only IPython and IPyKernel are required to launch a Python process as a kernel and execute code against your notebook, e.g., `pip install ipython ipykernel`. Visit our [wiki](https://github.com/microsoft/vscode-jupyter/wiki/Kernels-(Architecture)) to learn more.
```

Comment:
Not sure why we need to mention `IPython`, 
The only requirement is for the user to have `IPyKernel` module to be installed, hence i would not mention `IPython`

Reply:
Thanks, fixed :)